nvidia-gpu/nvidia-fabricmanager/lts/pkg.yaml

```diff
@@ -2,56 +2,93 @@ name: nvidia-fabricmanager-lts
 variant: scratch
 shell: /bin/bash
 dependencies:
- - stage: base
+  - stage: base
+    from: /
+    to: /base-rootfs
+  - image: cgr.dev/chainguard/wolfi-base@{{ .WOLFI_BASE_REF }}
```

Comment:
Agreed, this shouldn't need Wolfi IIUC, since no Glibc build happens here

Reply:
It looks much better indeed. Let me know what you think I wasn't sure how to name it.
I'm satisfied with how it builds so far. Once this round of review is cleared I'll schedule an upgrade for validation.

Reply:
Ok that was a completely unrelated issue with [talhelper](https://github.com/budimanjojo/talhelper).
We have been using this talhelper config for a while:
```yaml
    machineFiles:
      - op: create
        content: |
          [plugins]
            [plugins."io.containerd.cri.v1.runtime"]
              [plugins."io.containerd.cri.v1.runtime".containerd]
                default_runtime_name = "nvidia"
        path: /etc/cri/conf.d/20-customization.part
```
Turns out it has been generating a `machine.file:` entry with `permissions: 0o0`, which is bad but used to work. Now it breaks on 1.10.6, so `permissions: 0o444` has to be set explicitly with talhelper.

Reply:
so is this good to go?

Reply:
It failed again. I don't get it, looks like I can't make working builds anymore.

Previously I never had to call the imager with `--system-extension-image ghcr.io/siderolabs/glibc:x`. Now if I don't, then the service container fails to bind mount `/usr/local/glibc` because it doesn't exist.
But then, when explicitely imaging with the glibc extention, then I still get errors I had never see before about libraries such as :
```
nv-fabricmanager: /usr/bin/nv-fabricmanager: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory
```

Reply:
Sorry about the noise, I figured it out. Silly problem on my side.

So it's good to go, I validated the current PR head.
- rebased on 1.10.6 : OK and now used in production
- rebased on release-1.11 : OK too